app/views/schools/manual_readings/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1><%= t('manage_school_menu.manage_manual_readings') %></h1>
+<% if @current_target.nil? %>
+  <p><%= t('schools.manual_readings.set_target_html', path: new_school_school_target_path(@school)) %></p>
+<% elsif @months.values.flatten.empty? %>
+  <p><%= t('schools.manual_readings.enough_data') %></p>
+<% else %>
+  <p><%= t('schools.manual_readings.intro') %></p>
+  <% if @current_target %>
+    <p><%= t('schools.manual_readings.completed', start_date: @current_target.start_date.to_fs(:es_long)) %></p>
+  <% end %>
+  <%= simple_form_for @school, url: [@school, :manual_readings] do |f| %>
+    <table>
+      <thead>
+        <tr>
+          <th><%= t('common.labels.date') %></th>
+          <% @months.each do |fuel_type, months| %>
+            <th><%= t("common.#{fuel_type}") unless months.empty? %></th>
+          <% end %>
+        </tr>
+      </thead>
+      <tbody>
+        <%= f.simple_fields_for :manual_readings, @readings do |reading_form| %>
```

Comment:
Fields need to allow for float inputs. Usage might be 17.5 kWh for example. 

Also don't use numeric fields, as the UX adds the controls for increasing and decreasing by one, which isn't great for data entry. Users are rarely going to use those.